src/foam/u2/view/RadioEnumView.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Wrong copyright notice. Should be:

/**
 * @license
 * Copyright 2022 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */